rustls/src/msgs/handshake.rs

```diff
@@ -1448,10 +1448,10 @@ impl CertificatePayloadTls13 {
             .unwrap_or_default()
     }
 
-    pub(crate) fn convert(&self) -> CertificatePayload {
+    pub(crate) fn convert(self) -> CertificatePayload {
         let mut ret = Vec::new();
```

Comment:
Drive by review: this function could make an iterator out of `self.entries`, `map` `entry.cert` and `collect`?